### PR TITLE
add entries to the napari-tools menu

### PR DESCRIPTION
**Pull request by haesleinhuepf**
Hi Johannes @jo-mueller ,

here comes a PR-draft that adds some of your functions ot the napari-tools menu. It's not functional in this state but I don't know enough about `frame-by-frame`. I have the suspicion that `frame-by-frame` does handle 3D data wrongly, but I'm not sure.

Happy to discuss tomorrow.

Best,
Robert

### CODE REVIEW

**Comment by haesleinhuepf** [comment]
Bug fixed. With this change, the menus from the tool-menu work also on single timepoints at frame > 0. You can then combine surface-processing functions between napari-stress and nppas

**Comment by jo-mueller** [comment]
Hm...I am still having the problem that points data created from nppas does not update over time :/ There aren't even napari-stress functions involved at this point.

**Comment by haesleinhuepf** [comment]
Ok, I'm done here. Ready for review @jo-mueller

**Comment by jo-mueller** [comment]
Works like a charm! I'll create an issue to point to the correct entries in the tools menu in the recipe for surface reconstruction 👍

File: src/napari_stress/_napari_plugin.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from ._surface import smooth_sinc
```
Comment (jo-mueller) [inline src/napari_stress/_napari_plugin.py:1]: what does this file do?

**Pull request merged**